libs/partners/ibm/pyproject.toml

```diff
@@ -22,7 +22,7 @@ optional = true
 pytest = "^7.3.0"
 freezegun = "^1.2.2"
 pytest-mock  = "^3.10.0"
-ibm-watsonx-ai = "^0.1.8"
```

Comment:
we don't actually need to specify ibm-watsonx-ai in any of the group dependency since it's already a hard dependency of the package